src/point.rs

```diff
@@ -1,45 +1,99 @@
-use dislog_hal::DisLogPoint;
+use core::ops::{Add, Mul, Neg};
```

Comment:
You can remove these traits import

Reply:
The modification has been submitted, all is fine